routes/web/project/location/load_balancer.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class CloverWeb
+  hash_branch(:project_location_prefix, "load-balancer") do |r|
+    r.on String do |lb_name|
+      pss = @project.private_subnets_dataset.where(location: @location).all
+      lb = pss.flat_map { _1.load_balancers_dataset.where { {Sequel[:load_balancer][:name] => lb_name} }.all }.first
```

Comment:
We already did similar change for the other part

```suggestion
      lb = @project.load_balancers_dataset.where(location: @location).where( {Sequel[:load_balancer][:name] => lb_name} ).first
```

Reply:
Yes but LoadBalancer does not have `location` it's location is resolved through private subnet.

Reply:
nit: This version generates fewer database queries and is more intuitive. However, it's totally personal. The decision is up to you.

```suggestion      
      lb = @project.load_balancers_dataset.where( {Sequel[:load_balancer][:name] => lb_name} ).all.find { _1.private_subnet.location == @location}
```